src/main/java/org/hyperskill/musicadvisor/commands/Auth.java

```diff
@@ -0,0 +1,25 @@
+package org.hyperskill.musicadvisor.commands;
+
+public class Auth {
+    private static final String clientId = "2cbc4ced003f4fc895a890f1b94ab6ff";
```

Comment:
The names of variables declared class constants should be all uppercase with words separated by underscores ("_")